gems.rb

```diff
@@ -24,4 +24,7 @@
 	gem "bake-test-external"
 	
 	gem "sqlite3", "~> 1.4"
+	gem "rails", "< 8.0"
+
+	gem 'console', git: 'git@github.com:HolyWalley/console.git', branch: 'support-tagged-logging'
```

Comment:
added it for now to make sure tests are passing, will replace with minimal version in gemspec

in terms of rails, it's been failing for me to use sqlite of version < 2.1. Don't have time to figure it out now